libs/layout/src/lib/components/configurable-layout/configurable-layout.component.ts

```diff
@@ -341,6 +364,32 @@ export class NgxConfigurableLayoutComponent
 		this.form.setValue(formData);
 	}
 
+	/**
+	 * Moves an element based on its provided event
+	 *
+	 * @param event - The provided move event
+	 */
+	public move(event: NgxAccessibleDragAndDropMoveEvent): void {
+		// Iben: Get the required data
+		const formData = [...this.form.value];
+		const startArray = formData[event.previousContainer];
+		const endArray = formData[event.newContainer];
+
+		// Iben: If the drag and drop is within the same row, we move
+		if (event.previousContainer === event.newContainer) {
+			moveItemInArray(endArray, event.previousIndex, event.newIndex);
+		} else {
+			// Iben: If the drag and drop is over multiple rows, we transfer
+			transferArrayItem(startArray, endArray, event.previousIndex, event.newIndex);
+			formData[event.previousContainer] = startArray;
```

Comment:
Just checking: Is the `previousContainer` always defined?

Reply:
It should be, but I'll double check here as well